zh_cn/02.安装与配置/03.helm.md

```diff
@@ -36,8 +36,9 @@ Zadig 支持两种访问方式，一种是提供访问域名，另一种是直
 > 请确保您有已备案的域名。
 
 ```bash
-helm upgrade --install zadig --namespace zadig  koderover-chart/zadig --set endpoint.FQDN=<your domain> 
-# v1.7.0+ 版本需加上以下参数
+helm upgrade --install zadig --namespace <NAMESPACE> koderover-chart/zadig --set endpoint.FQDN=<your domain>
+# v1.7.0+ 版本需加上以下参数，且确保 global.extensions.extAuth.extauthzServerRef.namespace 和安装 Zadig 系统的 Namespace 一致
+#   --set global.extensions.extAuth.extauthzServerRef.namespace=<NAMESPACE>
```

Comment:
For dex, the following parameters need to be added: `--set dex.fullnameOverride=<your domain>`

Reply:
better be fullnameOverride=zadig-{your_namespace}-dex for debugging purpose.